### PR TITLE
WordPress migration plugin

### PR DESCRIPTION
**Pull request by saisubodhMSFT**
Added quickstart documentation for migration plugin

### CODE REVIEW

File: WordPress/WordPress_Migration_Plugin.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Installation
+
+Follow these steps to install the Azure App Service Migration plugin:
+
+1. Open the WordPress dashboard of your site.
+
+2. In the dashboard menu, navigate to **Plugins > Add New** section.
+
+   ![Installation Step 2](./media/aasm_addPlugin.png)
```
Comment (ZubaeyrMSFT) [inline WordPress/WordPress_Migration_Plugin.md:21]: Incorrect filename - aasm_addplugin.png

File: WordPress/WordPress_Migration_Plugin.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+   ![Installation Step 3](./media/aasm_uploadPlugin.png)
+
+4. Click on **Install**.
+
+   ![Installation Step 4](./media/aasm_installPlugin.png)
+
+5. After installation, click on **Activate**.
+
+   ![Installation Step 5](./media/activatePlugin.png)
```
Comment (ZubaeyrMSFT) [inline WordPress/WordPress_Migration_Plugin.md:33]: Incorrect file name - aasm_activatePlugin.png

File: WordPress/WordPress_Migration_Plugin.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+- **PHP**: Version 7.0 or higher
+  - Ensure that the source site has PHP version 7.0 or higher.
+  - Note that the destination WordPress site on the Azure Linux App Service runs on PHP version 8.0 or higher.
+
+## Installation
+
+Follow these steps to install the Azure App Service Migration plugin:
+
+1. Open the WordPress dashboard of your site.
```
Comment (ZubaeyrMSFT) [inline WordPress/WordPress_Migration_Plugin.md:17]: Should you be adding step where the user can download the plugin from? the link to plugin repo or zip file?

Comment (saisubodhMSFT) [reply WordPress/WordPress_Migration_Plugin.md]: Done

**Review by ZubaeyrMSFT — APPROVED** [review]

**Pull request merged**